server/cassandraApp.js

```diff
@@ -0,0 +1,161 @@
+const express = require('express');
+const app = express();
+const cassie = require('../CassandraDB/index.js')
+const path = require('path');
+
+
+app.use(express.json())
+app.use('/products', express.static('./client/dist'))
+app.use('/', express.static('./client/dist'))
+
+
+// this function will find the highest id in the database that does not already have an entry. This 'highest' variable can be adjusted to narrow the
+// search down -- for example the database was explicitly seeded with 10,000,000 entries, so there's no need to look before that for unused ids
+let highest = 10000000;
+let searchForHighest = () => {
+  let tryHighest = cassie.execute(`SELECT * FROM mykea.items WHERE id = ${highest}`).then(results => results);

```

Comment:
curious why you need the .then (result => results) if you're not doing anything to the results before returning them again. I might be missing something but wouldn't it be less code do just say:
"let tryHighest = cassie.execute(`SELECT * FROM mykea.items WHERE id = ${highest}`)"
tryHighest.then((results) => { if (results.row[0]....?

Reply:
Good catch. That was also left over from a previous way of doing things. I will remove it!